en/docs/reference/streaming-connectors/downloading-and-installing-siddhi-extensions.md

```diff
@@ -159,7 +159,7 @@ To download the Siddhi extensions, follow the steps below
     ![Older Versions]({{base_path}}/assets/img/streaming/downloading-and-installing-siddhi-extensions/extensions-older-versions.png)
 
     You are directed to the maven central page where all the available versions of the extension are listed.  
-    ![All available versions]({{base_path}}/assets/img/streaming/downloading-and-installing-siddhi-extensions/central-maven-repository.png)
+    ![All available versions]({{base_path}}/assets/img/streaming/downloading-and-installing-siddhi-extensions/Central Maven Repository.png)
```

Comment:
This image will break when this page goes live as it has upper case in it.

Also as a practise to not have spaces in file names. Use hyphens instead of spaces.

Reply:
This is outdated I think. Can you refresh and check.